auth/index.html

```diff
@@ -503,19 +508,16 @@ <h2>Message log</h2>
         };
 
         /**
-         * Function to send a message. If this page is embedded as an iframe we'll use window.top.postMessage. Otherwise we'll display it in the DOM.
+         * Function to send a message. If this page is embedded as an iframe we'll use the MessageChannel port from the parent frame. Otherwise we'll display it in the DOM.
          * @param type message type. Can be "PUBLIC_KEY_CREATED", "BUNDLE_INJECTED" or "STAMP"
          * @param value message value
          */
         var sendMessageUp = function (type, value) {
-          if (window.top !== null) {
-            window.top.postMessage(
-              {
-                type: type,
-                value: value,
-              },
-              "*"
-            );
+          if (parentFrameMessageChannelPort) {
+            parentFrameMessageChannelPort.postMessage({
+              type: type,
+              value: value,
+            })
```

Comment:
for compatibility we should keep sending messages with `window.top.postMessage` (actually, we should probably fix this to be `window.parent.postMessage`). SDKs in the wild who are currently using Turnkey's iframes will need this to keep running smoothly.

A breaking change isn't an option unfortunately -- the other option at our disposal would be to version our deployment and host authv2.turnkey.com which points at the new code. But given the level of effort I think it's easy enough to have logic which says "if there is a channel to use, use it. Otherwise, talk to the parent page.

As SDKs and webpages update to the newer version of Turnkey SDK more and more iframes will use channels, and eventually (maybe) we can retire the code which uses "window.top.postMessage" entirely! But that will be months from now.

Reply:
I updated this to be `window.parent.postMessage` for backwards compatibility. As an aside, do we have tracking / analytics on which versions of our SDKs are live in the wild?